### DISCUSSION TITLE
specs-actors 不同版本MultisigActorCodeID 创建多签地址

### DISCUSSION THREAD

**Discussion by jc0803kevin** [filecoin-project/lotus#7554] [General Q&A] [answered]
specs-actors 不同版本MultisigActorCodeID 创建多签地址
fil/5/multisig
fil/6/multisig
fil/7/multisig

使用【fil/5/multisig】创建的多签地址，节点升级之后【fil/7/multisig】
在该版本原来历史版本创建的多签地址，提议，批准交易是否有影响??

(
specs-actors Different versions of MultisigActorCodeID create multi-signature addresses
fil/5/multisig
fil/6/multisig
fil/7/multisig

Use the multi-signature address created by [fil/5/multisig], after the node is upgraded [fil/7/multisig]
Does the multi-signature address, proposal, and approval transaction created in the original historical version of this version have any impact?)

**Comment by miles-zhangdy** [comment]
We have the same problem!!!!!!!!!

**Reply by tonyzhou1122** [reply]
We have the same problem!

**Comment by SgtCoin** [comment]
@miles-zhangdy @tonyzhou1122 

Can you share the addresses you are having trouble finding?

**Reply by miles-zhangdy** [reply]
t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i

**Reply by SgtCoin** [reply]
@miles-zhangdy 
I am able to find that address on the calibnet. 

`./lotus msig inspect t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i
Balance: 200.979 FIL
Spendable: 200.979 FIL
Threshold: 3 / 6
Signers:
ID       Address
t022393  t1xjovbleaxpeoecg3d6ciddb2bjhiztlglk2g7fa
t022735  t1y7ht7br7sasn5r6leakuiyejw3t44ilelmc77ta
t022736  t1dxfpauach23nubzlsosorjmmvw4asqaybrlpn6a
t022737  t1mjifcmgbqlfoa7qb4tvash53nabibv3chaioa5i
t022738  t1snelsnfgkttupe6d522feyneqhx4trpuco2h5ti
t022739  t13tnwzneqlkpaauwsth5yehqhhygsxy3hgebdqaa
Transactions:  0
trey@lab:/usr/local/bin/lotus$
`

#############

`/usr/local/bin/lotus./lotus state get-actor t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i
Address:        t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i
Balance:        200.979 FIL
Nonce:          0
Code:           bafkqadtgnfwc6nrpnv2wy5djonuwo (fil/6/multisig)
Head:           bafy2bzaced5evg73htjbqe2gv3ehdksgscvx2ekvszxrqcl37xrihe2pskrus
`
Can you elaborate more on the issue you are facing?

**Reply by jc0803kevin** [reply]
以这个地址 ：t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i为例，在多签版本升级到 【fil/7/multisig】这个的时候，那该地址在升级之后提议交易，批准交易是否有影响？？

(Take this address: t2l5hpmfuof7u2cxwxevvbdqfq4neoxjo4btcye3i as an example. When the multi-signature version is upgraded to [fil/7/multisig], then the address proposes a transaction after the upgrade. Does it affect the approval of the transaction? ?)

**Comment by SgtCoin** [comment, marked as answer]
It should not affect the approval as doing so (due to a change in version) would break all multisig addresses. We expect that the upgrade be non-disruptive to your deal making procedures.

**Reply by jc0803kevin** [reply]
好的，非常感谢！

(alright, thank you very much!)

**Reply by SgtCoin** [reply]
No problem! If you have any issues please let me know. I will be happy to help.
